docs/getting_started/SERVICES.md

```diff
@@ -0,0 +1,51 @@
+# Services
+
+Bolt provides a concept of _services_ to split up components of Rivet's architecture. See `lib/bolt/config/src/service.rs` to jump straight to the source.
```

Comment:
Within the grand scheme of Rivet, does a service mean a unit that is compiled into a final full Rivet binary, or a microservice that is being run on the infrastructure? I think describing that would be a good preamble for how the rest of the sections below are interpreted.

Reply:
fcd61c91